### ISSUE TITLE
⌨ Assessment decision dialog table header names not programmatically discernable

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#9380]
## ⌨ Accessibility Issue

The icon only headers are causing some issues since they cannot be discerned by the accessibility tree.

### WCAG 2.1 Criteria

 - https://dequeuniversity.com/rules/axe/4.8/empty-table-header

## 🦋 Expected Behaviour

The accessibility tree should be able to determine the name of the table headers.

## 📋 Steps to Reproduce

1. Login as admin
2. Navigate to an application
3. Open the final descision dialog
4. Inspect the table headers in the accessibility tree
5. Observe they have no name

## 📸 Screenshot

![Screenshot 2024-02-15 132712](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/84231f00-c16c-4d12-a538-f557fb4002b4)

## 🙋‍♀️ Proposed Solution

We have the `aria-label`'s there, the problem is they are on `svg` which are hidden to the tree by default. You should just need to add `aria-hidden="false"` to them or move the label to the `th` instead.

## ✅ Acceptance Criteria

- [ ] Final decision dialog table headers have a name in the accessibility tree
